### ISSUE TITLE
Tooltip is not well positioned in inlineMode

### ISSUE THREAD

**Issue by roipoussiere** [wakirin/Litepicker#100]
**Litepicker Version**

v1.4.5

**Describe the bug**
Tooltip is not well positioned in inlineMode.

**To Reproduce**

When using this simple html:
```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <script type="application/javascript" src="/static/js/vendor/lite-picker-with-css.min.js"></script>
</head>
<body>
<input id="litepicker"/>
<script>
var date_picker = new Litepicker({
  element: document.getElementById('litepicker'),
  inlineMode: false,
  singleMode: false,
  });
</script>
</body>
```

it works well:

![2020-06-05_09-06-1591341154](https://user-images.githubusercontent.com/1665542/83848504-4491ad80-a70e-11ea-97eb-b1a95ab3976b.jpg)

But when setting `inlineMode` to `false`, the tooltip is not well positioned:

![2020-06-05_09-06-1591341192](https://user-images.githubusercontent.com/1665542/83848646-79056980-a70e-11ea-9118-72d560617563.jpg)

**Expected behavior**

The tooltip position should be the same, whatever the `inlineMode` value.

**Comment by wakirin** [comment]
Fixed in v1.5.2
https://jsfiddle.net/waki/04w6ombp/2/

**Comment by wakirin** [comment]
Try to use with `parentEl`.

https://jsfiddle.net/waki/8pujt5rv/

**Comment by ninetyninew** [comment]
Had this same issue - @wakirin's comment fixes this for me - thanks.

**Comment by wakirin** [comment]
fixed in 2.0.4
